.github/workflows/google-deploy.yaml

```diff
@@ -9,6 +9,8 @@ on:
 
 env:
   BUCKET_NAME: sso-dashboard-${{ inputs.environment }}
+  CHANNEL_IDS: C05AMLCL4JX
```

Comment:
No, it shouldn't be secret.  It is treated the same way we treat auth0 client IDs.  The secret is they oauth key which is already set as a secret.  It also helps when debugging as you can change the channel id on the fly as opposed to changing it repo or org wide.

I do agree this could use a comment as to which slack channel this attaches to.  I'll add it.